sheet.go

```diff
@@ -85,6 +85,10 @@ func (f *File) workbookReader() *xlsxWorkbook {
 		var content xlsxWorkbook
 		_ = xml.Unmarshal(namespaceStrictToTransitional(f.readXML("xl/workbook.xml")), &content)
 		f.WorkBook = &content
+		//rebuild sheetID
```

Comment:
If the ID associated with it is not updated together，this modification will lead to the corruption of the spreadsheet in some cases. Ref. [ECMA-376 5th edition Part 1 §18.11.1.23](https://www.ecma-international.org/publications/standards/Ecma-376.htm)